llvm/test/CodeGen/X86/vector-shuffle-512-v64.ll

```diff
@@ -109,13 +109,13 @@ define <64 x i8> @shuffle_v64i8_79_00_01_02_03_04_05_06_07_08_09_10_11_12_13_14_
 define <64 x i8> @shuffle_v64i8_0zzzzzzzzzzzzzzzzzzzzzzzzzzzzzzz(<64 x i8> %a) {
 ; AVX512F-LABEL: shuffle_v64i8_0zzzzzzzzzzzzzzzzzzzzzzzzzzzzzzz:
 ; AVX512F:       # %bb.0:
-; AVX512F-NEXT:    vmovd {{.*#+}} xmm1 = [255,0,0,0]
```

Comment:
Shouldn't we prefer `vmovd` to `vpmovzxbq`?

Reply:
I guess it comes down to whether we want to save those 16-bits in the constant pool in exchange for a shuffle uop

Reply:
I see, I forgot the size difference.